package.json

```diff
@@ -6,6 +6,10 @@
   ],
   "scripts": {
     "build": "lerna run build",
+    "build:visual": "docker build -t screenshot:latest ./;",
+    "test:visual": "yarn build:visual && docker run -t screenshot:latest yarn test:integration",
```

Comment:
We do need a way to generate initial screenshots, locally just running `yarn test` with empty `__image_snapshots__` generates the files.

Also need to be able to update screenshots with `yarn test:updateScreenshots` or something - locally, `yarn test -u` accomplishes that.

Reply:
>We do need a way to generate initial screenshots, locally just running yarn test with empty __image_snapshots__ generates the files.

Not sure what you mean here. `__image_snapshots__` is tracked in git, so in any environment (local or Travis) will have an __image_snapshots__ directory.

>Also need to be able to update screenshots with yarn test:updateScreenshots or something - locally, yarn test -u accomplishes that.

The last two commands below this are `yarn test` and `yarn test:integration`:

https://github.com/meetup/swarm-ui/pull/72/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R13

To update you don't need a separate command, just append `-u`, like `yarn test:integration -u` at the root level or within `packages/swarm-components`. This is true in the current master.

Reply:
These commands run locally, not in Docker so they generate screenshots using developer computer's Chrome and not the same rendering engine that Docker version will use which will cause tests to fail.

We need Docker version to always be ran, for initial generation, for screenshot updates if developer made a change and needs a new screenshot generated and for actual testing.

Reply:
Sorry, I should have clarified. I meant you can run those yarn commands inside the docker container. I forget the exact command - but you can do the equivalent of ssh-ing into the container and run the yarn commands there and then use the copy command to move it to the local filesystem.

Or perhaps there is a `docker run` equivalent with something like `docker run yarn test:integration -u` with the correct container options passed in, but I'm not 100% sure that's possible. 

tl;dr those yarn commands already exist, we just need to document the correct docker steps.

Reply:
we now have 2 `make` commands to

1. run tests: `make test-visual`
2. update the local screenshots: `make test-visual-update`

Both commands update the local filesystem with screenshot changes generated inside the container